content/docs/apps/static-egress.md

```diff
@@ -5,15 +5,22 @@ menu:
 title: Communicating with external services
 ---
 
-If your cloud.gov-hosted application needs access to an external service, that service must be accessible over the internet. All traffic from cloud.gov applications will transit the internet.
+You can set up your cloud.gov application to communicate with any external service that is accessible over the internet. For example, you can integrate your application with services in your agency infrastructure or with APIs from commercial providers.
```

Comment:
I don't like the phrase 'cloud.gov application'. Applications are hosted on cloud.gov, but are otherwise independent entities.

Reply:
I see how that's ambiguous! I revised it for clarity.